tests/unit/test_buyers_helpers.py

```diff
@@ -74,6 +74,29 @@ def test_brief_can_be_edited(self):
         assert helpers.buyers_helpers.brief_can_be_edited(api_stubs.brief(status='draft')['briefs']) is True
         assert helpers.buyers_helpers.brief_can_be_edited(api_stubs.brief(status='live')['briefs']) is False
 
+    def test_section_has_at_least_one_required_question(self):
+        brief = {
+            'status': 'draft',
+            'frameworkSlug': 'dos',
+            'lotSlug': 'digital-specialists',
+            'required1': True
+            }
+        content = content_loader.get_manifest('dos', 'edit_brief').filter(
```

Comment:
It would be nice to use a test manifest for this rather than the actual DOS one... but that's probably a bigger job for another time.